.travis.yml

```diff
@@ -14,16 +18,9 @@ addons:
       - libboost-date-time-dev 
       - libboost-test-dev
       - google-mock
- 
-before_script:
-  - "cd /usr/src/gmock && sudo cmake . && sudo cmake --build . && sudo mv libg* /usr/local/lib/ && sudo mv gtest/libg* /usr/local/lib/ ; cd -"
 
 script: ./travis.sh 
 
-branches:
-  only:
-    - master
-
```

Comment:
Because it stopped my branch from building on travis. It's irrelevant to this repository as it hasn't got any active branches.